skrf/networkSet.py

```diff
@@ -800,8 +800,10 @@ def uncertainty_ntwk_triplet(self, attribute: str, n_deviations: int = 3):
         ntwk_std = self.__getattribute__('std_'+attribute)
         ntwk_std.s = n_deviations * ntwk_std.s
 
-        upper_bound = (ntwk_mean + ntwk_std)
-        lower_bound = (ntwk_mean - ntwk_std)
+        upper_bound = ntwk_mean.copy()
+        lower_bound = ntwk_mean.copy()
+        upper_bound.s = (ntwk_mean.s + ntwk_std.s)
```

Comment:
I don't understand why you see a difference with this and the previous implementation. Adding two Networks do:

https://github.com/scikit-rf/scikit-rf/blob/d1f60e701cac7276784510a4aaba74a68cc0f0a5/skrf/network.py#L581-L594

which looks like similar to what you propose. Do have some examples ?

Reply:
Apparently I was looking at a bad set of data. I'll find something better to present.

Reply:
I think the solution here is to create a method to optionally extract the minimums and maximums from the upper_bound and lower_bound network objects. This could create two new objects containing the minimum dB values and maximum dB values for each frequency.

Reply:
Apparently the issue was that I was calling `uncertainty_ntwk_triplet('s')` instead of `uncertainty_ntwk_triplet('s_mag')`. I'm all set.